main.py

```diff
@@ -15,16 +13,19 @@
 user_list = [x.strip() for x in os.environ.get("USER_LIST", "").split(",")]
 duration = int(os.environ.get("DURATION_IN_DAYS", 5))
 data_count = int(os.environ.get("PAGE_DATA_COUNT", 5))
+start_date = os.environ.get("START_DATE", "2023-10-01T00:00:00")
+
 
 variables = {
-    "timedelta": get_date(duration),
+    "timedelta": start_date,
     "dataCount": data_count,
 }
 
 
 def main():
     """Script entrypoint."""
 
+    print(variables)
```

Comment:
Do we need to print out variables? or is it missed?

Reply:
Missed, will change it